regenerate-initrd-posttrans

```diff
@@ -55,8 +48,7 @@ for f in "$dir"/*; do
 		echo "$0: /boot/$image does not exist, initrd won't be generated"
 		continue
 	fi
-	run_mkinitrd_setup
-	if ! /sbin/mkinitrd -k "/boot/$image" -i "/boot/$initrd"; then
+	if ! "$DRACUT" -f "/boot/$initrd"; then
```

Comment:
It turns out that this is wrong. I was under the impression that dracut would "guess" the kernel from the initrd file name, but it doesn't. If the kernel version isn't specified, it will generate the initrd for the *currently running kernel*, which is of course very wrong here.

Reply:
Solved in 16.0.2